nomad-cluster-security.md

```diff
@@ -0,0 +1,14 @@
+# Nomad Cluster Security
+
+1. What is the noamd cluster size?
+1. Is Nomad deployed single region or multi-region? What is the deployment strategy? How is the high availability and failover designed? How is leader and follower selected?
```

Comment:
Break this into sub-questions, please!

Reply:
Thank you for feedback . I have broke this into sub-questions like this .

Deployment Strategy

   - Is Nomad deployed in a single region or multi-region?
   - How is the high availability and failover designed?
   - In case of failover , how is the leader and follower selected?
   - How does the brain-split strategy work ?